### ISSUE TITLE
Deprecated warning

### ISSUE THREAD

**Issue by tchret** [joshwnj/react-visibility-sensor#28]
Hey @joshwnj, 

First, thanks for this usefull component :+1: 
I got this warning:

```
Warning: VisibilitySensor.getDOMNode(...) is deprecated. Please use ReactDOM.findDOMNode(instance) instead.
```

**Comment by AntonTrollback** [comment]
+1

**Comment by joshwnj** [comment]
@tchret looks like that was fixed but an old version remains in https://github.com/joshwnj/react-visibility-sensor/blob/master/dist/visibility-sensor.js

This will be fixed in #33
